script/1_0_2current_en_script/Ciel/Day 4/04_01_CIEL04_6_2.txt

```diff
@@ -76,10 +76,10 @@ I bow my head sincerely.
 "A-Anyway, you are too vulnerable. The enemy is a vampire, so you've got to be more careful."
 <Page24>
 "Or what, do you have a death wish or something? Are you trying to get yourself killed or to piss me off?
-If so, I can repeatedly go from my Fundamental State to my Excited State in a flash. Why don't I throw you into a space so absolute that human speech can't describe it?"
+If that's the case, I can jump from my ground state to an excited state at the drop of a hat. Would you like me to knock you into a spatial extreme that's beyond the limit of human comprehension?
```

Comment:
Ground/excited state here reads to me like a reference to orbitals, rather than some nasuverse term, so not sure we need caps on those. I think that limit/extreme is maybe a better word for 極限 than absolute, since I think it's meaning can be mathematical limit (inkeeping with our `science` theme). There may be a better jargon word than spatial extreme here though.

> そういうコトなら基底状態と励起状態をコンマで繰り返す、人間の言葉じゃ表現できないぐらいの極限空間にたたき込んであげましょうか？」

Reply:
I agree on both counts. Absolute space is indeed a legitimate term but spatial extreme definitely reads better to me